client/docker/docker-compose.yml

```diff
@@ -41,14 +31,7 @@ services:
       - data01:/usr/share/elasticsearch/data
     ports:
       - 9200:9200
-    # networks:
-    #   - opencr
-
 
 volumes:
   data01:
```

Comment:
Perhaps rename this volume to something more suitable while we make these other changes.